contribs/gmf/src/directives/layertree.js

```diff
@@ -609,6 +610,21 @@ gmf.LayertreeController.prototype.toggleActive = function(treeCtrl) {
 
 
 /**
+ * Search if a substring representing layers names are in the string representing
+ * all the layers names of the WM(T)S layer
+ * @param  {string} allLayersNames all layers names currently enable for the
+ * WM(T)S layer
+ * @param  {string} subLayersNames layers names to search on allLayersNames
+ * @return {boolean} true if subLayersNames is in allLayersNames
+ * @private
+ */
+gmf.LayertreeController.prototype.searchLayersNames_ = function(allLayersNames,
+  subLayersNames) {
+  var pattern = new RegExp(subLayersNames + '(?=\,|$)');
+  return allLayersNames.search(pattern) >= 0;
```

Comment:
Will not work if we have for example two layers names `a.b`, `a_b`...
Why not doing `allLayersNames.split(',').indexOf(subLayersNames) >= 0;`?